src/it/java/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/LineLengthTest.java

```diff
@@ -29,20 +29,14 @@ public class LineLengthTest extends AbstractGoogleModuleTestSupport {
 
     @Override
     protected String getPackageLocation() {
-        return "com/google/checkstyle/test/chapter3filestructure/rule32packagestate";
+        return "com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap";
     }
 
     @Test
     public void testLineLength() throws Exception {
         final String[] expected = {
-            "5: " + getCheckMessage(LineLengthCheck.class, "maxLineLen", 100, 112),
-            "29: " + getCheckMessage(LineLengthCheck.class, "maxLineLen", 100, 183),
-            "46: " + getCheckMessage(LineLengthCheck.class, "maxLineLen", 100, 131),
-            "47: " + getCheckMessage(LineLengthCheck.class, "maxLineLen", 100, 124),
-            "48: " + getCheckMessage(LineLengthCheck.class, "maxLineLen", 100, 113),
-            "50: " + getCheckMessage(LineLengthCheck.class, "maxLineLen", 100, 116),
-            "53: " + getCheckMessage(LineLengthCheck.class, "maxLineLen", 100, 131),
-            "57: " + getCheckMessage(LineLengthCheck.class, "maxLineLen", 100, 116),
+            "7: " + getCheckMessage(LineLengthCheck.class, "maxLineLen", 100, 112),
+            "9: " + getCheckMessage(LineLengthCheck.class, "maxLineLen", 100, 113),
```

Comment:
this array is not required if do migration.
What force you yo keep it?

Reply:
this test file is of section 3.3.2 and I was only focusing on chapter 3.2 for this PR. I thought it would be better to transition the chapter 3.3.2 in its separate PR instead mixing everything into one PR

Reply:
Ok